db/migrations/admin-migrations/2023-10-16.0.add-tenants-table.sql

```diff
@@ -33,20 +33,18 @@ CREATE TABLE public.tenants
     updated_at TIMESTAMP WITH TIME ZONE NOT NULL DEFAULT NOW()
 );
 
-CREATE UNIQUE INDEX idx_unique_name ON public.tenants (LOWER(name));
-CREATE TRIGGER refresh_tenants_updated_at BEFORE UPDATE ON public.tenants FOR EACH ROW EXECUTE PROCEDURE update_at_refresh();
+CREATE UNIQUE INDEX idx_unique_name ON tenants (LOWER(name));
+CREATE TRIGGER refresh_tenants_updated_at BEFORE UPDATE ON tenants FOR EACH ROW EXECUTE PROCEDURE update_at_refresh();
 
-COMMENT ON COLUMN public.tenants.base_url IS 'The SDP backend server''s base URL';
-COMMENT ON COLUMN public.tenants.sdp_ui_base_url IS 'The SDP UI/dashboard Base URL.';
+COMMENT ON COLUMN tenants.base_url IS 'The SDP backend server''s base URL';
+COMMENT ON COLUMN tenants.sdp_ui_base_url IS 'The SDP UI/dashboard Base URL.';
 
 -- +migrate Down
 
-DROP TABLE public.tenants;
+DROP TABLE tenants;
 
-DROP TYPE public.email_sender_type;
+DROP TYPE email_sender_type;
 
-DROP TYPE public.sms_sender_type;
+DROP TYPE sms_sender_type;
 
-DROP TYPE public.tenant_status;
-
-DROP FUNCTION update_at_refresh;
```

Comment:
Are we removing the `DROP` statement because `update_at_refresh` is used by other tables ?

Reply:
Yes!